search/src/main/java/edu/artic/search/SearchResultCellViewModels.kt

```diff
@@ -85,7 +85,8 @@ class SearchTourCellViewModel(val articTour: ArticTour) : SearchBaseListItemView
 /**
  * ViewModel for displaying the circular artwork image under "On the map" section.
  */
-class SearchCircularCellViewModel(val artwork: ArticObject?) : SearchBaseCellViewModel() {
+class SearchCircularCellViewModel(val artwork: ArticObject?,
+                                  override val order: Int = -1) : SearchBaseCellViewModel(), OrderedCellViewModel {
```

Comment:
I think we can remove the default value for `order` here. That way the compiler will error out if we forget to include a number.